packages/x-components/src/x-modules/identifier-results/store/__tests__/actions.spec.ts

```diff
@@ -53,6 +53,20 @@ describe('testing identifier results module actions', () => {
   });
 
   describe('fetchAndSaveIdentifierResults', () => {
+    it('should include the origin in the request', async () => {
+      resetIdentifierResultsStateWith(store, { query: 'xc', origin: 'search_box:external' });
+      const spiedSearchById = jest.spyOn(adapter, 'searchById');
+      await store.dispatch('fetchAndSaveIdentifierResults', store.getters.identifierResultsRequest);
+
+      expect(spiedSearchById).toHaveBeenCalledTimes(1);
+      expect(spiedSearchById).toHaveBeenCalledWith({
+        ...store.getters.identifierResultsRequest,
+        origin: 'search_box:external'
+      });
+      jest.clearAllMocks();
```

Comment:
Even thought right now we only have one test with mocks  I'd move this to the `beforeEach` just in case we'll add more tests with mocks.